monGARS/api/web_api.py

```
@@ -1,3 +1,48 @@
+
+from monGARS.api.authentication import get_current_user
+from monGARS.api.dependencies import get_hippocampus
+from monGARS.core.hippocampus import MemoryItem
+from monGARS.core.security import SecurityManager
+
+app = FastAPI(title="monGARS API")
+sec_manager = SecurityManager()
+
+
+@app.post("/token")
+async def login(form_data: OAuth2PasswordRequestForm = Depends()) -> dict:
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)